docs/developer/app-store/apps/segment.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Twilio Segment
+sidebar_position: 3
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+<AppMetadata
+  minSaleorVersion="3.14"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Segment"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/segment"
+/>
+
+## Introduction
+
+This app integrates [Twilio Segment](https://segment.com/) with Saleor. The app will send events to Segment as a source.
+
+## Features
+
+- Send order-related events to Segment
+- Use Segment [semantic e-commerce events](https://segment.com/docs/connections/spec/ecommerce/v2/#core-ordering-overview)
+
+## Assumptions & limitations
+
+- The app requires a [write key](https://segment.com/docs/connections/find-writekey/) to work.
+- The app is not validating if the write key is correct due to missing API on the Segment side. If the write key is incorrect, app will not inform you about it.
+
+## Permissions
+
+The app requires `MANAGE_ORDERS` permission to listen to order events.
+
+To save app configuration, staff users must have `MANAGE_APPS` permission.
+
+## Application flow
+
+The app uses the following events and maps them to Segment tracking events:
+
+| Saleor Event       | Segment event type |
+| ------------------ | ------------------ |
+| `ORDER_CREATED`    | "Checkout Started" |
+| `ORDER_FULLY_PAID` | "Order Completed"  |
+| `ORDER_REFUNDED`   | "Order Refunded"   |
+| `ORDER_UPDATED`    | "Order Updated"    |
+| `ORDER_CANCELLED`  | "Order Cancelled"  |
+
+The event will contain properties mapped from the following GraphQL fragment:
+
+```graphql
+fragment OrderBase on Order {
+  id
+  user {
+    id
+    email
+  }
+  channel {
+    id
+    slug
+    name
+  }
+  userEmail
+  shippingMethodName
+  total {
+    gross {
+      amount
+      currency
+    }
+    net {
+      currency
+      amount
+    }
+  }
+  lines {
+    id
+    productVariantId
+    productSku
+    variantName
+  }
+  number
+}
+```
+
+## Pre-requisites
+
+1. Create a [Segment account](https://app.segment.com)
+2. Create a new connection source and set it to `Node.js`.
```

Comment:
**suggestion (non-blocking)**:

If we can, I suggest linking to some Segment docs here.